.github/workflows/nightly.yaml

```diff
@@ -0,0 +1,69 @@
+name: Nightly OpsCenter Tests
+
+on:
+  workflow_dispatch:
```

Comment:
Not sure we should allow general public to start this job on demand. I remember discussing this with Josh many month ago and he did not want to do it. Please let me know if we should remove this ability.